### PR TITLE
Small icon redesign

### PR DESCRIPTION
**Pull request by christianlomboy**
## Motivation

Issue #14 

## Implementation

Functioning Flag and Like buttons, beginning work on Share button

### CODE REVIEW

**Comment by christianlomboy** [comment]
Still have to add ability to unlike a question

File: client/src/components/FlagButton.js [hunk 1]
```diff
@@ -0,0 +1,65 @@
...
+  constructor(props) {
+    super(props);
+    this.state = {
+      selected: 0,
+    };
+  }
+
+  componentDidMount() {
+    const svgToChangeColor = this.itemToChangeColor;
+    svgToChangeColor.addEventListener('colorchangeend', this.colorChangeDone);
```
Comment (aviskarkc10) [inline client/src/components/FlagButton.js:14]: let's use a name like `color-changed` or `colorChanged`. Will make it more readable. Also the variable name has a typo

File: client/src/components/FlagButton.js [hunk 2]
```diff
@@ -0,0 +1,65 @@
...
+        selected: nextProps.selected,
+      };
+    }
+  }
+
+  render() {
+    return (
+      <div
+        style={{
+          position: 'relative',
```
Comment (aviskarkc10) [inline client/src/components/FlagButton.js:37]: better to use class names than inline styling

File: client/src/components/LikeButton.js [hunk 3]
```diff
@@ -0,0 +1,74 @@
+import React from 'react';
```
Comment (aviskarkc10) [inline client/src/components/LikeButton.js:1]: looks like we can reuse so much f these logic using higher order components. The code in this file and in `FlagButton` are pretty much the same

Comment (christianlomboy) [reply client/src/components/LikeButton.js:1]: Okay, resolving first two suggestions and working on this one now

**Pull request merged**